src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { patchVotesByArticleId } from "../api";
+
+function ArticleVotes({ article, currentVotes }) {
+    
+  const [votes, setVotes] = useState(0);
+  const [error, setError] = useState(null);
+  const [optimisticMsg, setOptimisitcMsg] = useState(null);
+
+  const article_id = article.article_id;
+
+  const handleVote = (vote) => {
+    setVotes(votes + vote);
+    patchVotesByArticleId(article_id, vote)
+      .then(() => {
+        setOptimisitcMsg("Vote Submitted!");
```

Comment:
This message can be a bit confusing/misleading when the user changes their vote after first submitting it